index.html

```diff
@@ -0,0 +1,203 @@
+              <a href="#">Services</a>
+            </li>
+            <li>
+              <a href="#">Contact</a>
+            </li>
+          </ul>
+        </nav>
+        <!-- computer nav -->
+        <nav class="top-navigation fonts">
+          <div class="accounts">
+            <ul class="accounts-lists">
+              <li>
+                <a class="accounts-list" href="#"
+                  ><i class="fa fa-facebook"></i
+                ></a>
+              </li>
+              <li>
+                <a class="accounts-list" href="#"
+                  ><i class="fa fa-twitter"></i
+                ></a>
+              </li>
+              <li><a class="accounts-list" href="#">English</a></li>
+              <li><a class="accounts-list" href="#">My page</a></li>
+              <li><a class="accounts-list" href="#">Logout</a></li>
+            </ul>
+          </div>
+          <div class="navs">
+            <div class="nav-log">
+              <a href="index.html"
+                ><img class="nav-logo" src="images/dev-logo.png"
+              /></a>
+            </div>
+            <ul class="nav-lists">
+              <li><a class="nav-list" href="about.html">About</a></li>
+              <li><a class="nav-list" href="#">Projects</a></li>
+              <li><a class="nav-list" href="#">Services</a></li>
+              <li><a class="nav-list" href="#">Contact</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <!-- main section -->
+      <section class="home-page fonts">
+        <p class="hello">"Impossible is nothing!"</p>
+        <h1 class="web-title fonts-coco">
+          RMsoft.ltd - Reliable Made Software
+        </h1>
+        <div class="h-content">
+          <p>
+            RMsoft.ltd company (Reliable Made Software) founded in 2012 by a
+            team of professionals who are passionate about delivering the best
+            web based software and android applications
+          </p>
+        </div>
+        <h2 class="dates fonts-coco">Impossible is nothing</h2>
+        <p class="h-copyright">
+          Improving people’s work through reliable softwares necessary in their
+          activities
+        </p>
+      </section>
+    </main>
+    <!-- Main program -->
+    <section class="main-program fonts">
+      <div class="p-title">
+        <h2>SERVICES</h2>
+        <hr class="line" />
+      </div>
+
+      <div class="programs">
+        <!-- program 1 -->
+        <div class="program">
+          <div class="p-header">
+            <i class="fa services fa-desktop"></i>
+            <h4>Website design</h4>
+          </div>
+          <p class="p-text">
+            You need more than just a website. You need results. We turn
+            underperforming websites around. We have the business acumen,
+            creativity, and experience
+          </p>
+        </div>
+        <!-- program 2 -->
+        <div class="program">
+          <div class="p-header">
```

Comment:
- [ ] Kindly ensure to implement the change in `border-color` on `hover to the service inner containers, so as to ensure a very close pixel-perfect design.

- Behance design
<img width="460" alt="right" src="https://user-images.githubusercontent.com/41163571/225401072-abcf6278-bc01-455a-8839-6682841b84e6.png">

- Your design
![wrong](https://user-images.githubusercontent.com/41163571/225401454-e61bc62d-dce8-48c8-8daa-0bbf6fafc8bc.gif)